<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>testForClient</class>
 <widget class="QDialog" name="testForClient">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>429</width>
    <height>241</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>431</width>
     <height>241</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="labelQuestions">
      <property name="text">
       <string>Вопрос.</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="radioButtonNioigda">
        <property name="text">
         <string>Никогда/почти никогда</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButtonRedko">
        <property name="text">
         <string>Редко</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButtonChasto">
        <property name="text">
         <string>Часто</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButtonPostoyanno">
        <property name="text">
         <string>Постоянно/очень часто</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushAnswer">
        <property name="text">
         <string>Ответить.</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
